### ISSUE TITLE
Mod Suggestion: Dungeon Tactics

### ISSUE THREAD

**Issue by KosherKingOfIsrael** [xJon/Hexxit-II#154]
I would really like it if Dungeon Tactics returned for Hexxit II. I'm really disappointed that it hasn't, and after replaying the original modpack it feels missing.

**Comment by xJon** [comment]
Was Dungeon Tactics in the original Hexxit?

**Comment by KosherKingOfIsrael** [comment]
> Was Dungeon Tactics in the original Hexxit?

Yes it was.

**Comment by KosherKingOfIsrael** [comment]
> > Was Dungeon Tactics in the original Hexxit?
> 
> Yes it was.

I actually may be mistaken, I'm reviewing Hexxit's mod list and I don't see it. I distinctly remember seeing DT's items in it though, or items similar to what  DT has to offer. I'll do some more snooping.

**Comment by xJon** [comment]
Perhaps it is in the original Hexxit's alternative branch?
![image](https://user-images.githubusercontent.com/8945347/171020755-b49b6a0a-01cb-4098-8566-2179456aa9d2.png)

**Comment by KosherKingOfIsrael** [comment]
> Perhaps it is in the original Hexxit's alternative branch? ![image](https://user-images.githubusercontent.com/8945347/171020755-b49b6a0a-01cb-4098-8566-2179456aa9d2.png)

Good idea, I'll go through and see if any of the alternatives have it.

**Comment by BulletPR00FXX** [comment]
good mod, fun exploration.
